conf/etc/v2/components/csm.json

```diff
@@ -0,0 +1,161 @@
+{
+    "csm-agent": {
```

Comment:
Is this file for test only ?
if in test we want to keep separate from main code

can we create separate dir
test/conf/    ?

Reply:
I was considering these files as drafts for real component files. So, they are not only for tests.
Only limited amount of information is read for validation so far. The configuration provided in these files is severely incomplete (constrains are missing at least).